LibVLCSharp/Shared/RendererDiscoverer.cs

```diff
@@ -76,36 +76,34 @@ public class RendererItem : Internal
     {
         const int VideoRenderer = 0x0002;
         const int AudioRenderer = 0x0001;
+#if NETSTANDARD1_1
+        readonly Utf8StringMarshaler _utf8Marshaler = Utf8StringMarshaler.GetInstance();
```

Comment:
Can we just change the return type for all platforms to avoid the if?

Reply:
Not if we plan to use it as part of `DllImport` statements in which case it needs to be `ICustomMarshaler`

Reply:
`Utf8StringMarshaler` is a `ICustomMarshaler` anyway. You will only need a cast.